### DISCUSSION TITLE
BUILD FAILED (macOS 13.5.2 using ruby-build 20230904)

### DISCUSSION THREAD

**Discussion by dereknovak** [rbenv/ruby-build#2260] [Build failures] [answered]
I'm fairly new to the CLI and programming, so forgive me if I'm missing something obvious.

...

Last 10 log lines:
issue on GitHub <https://github.com/openssl/openssl/issues>
and include the output from the following command:

perl configdata.pm --dump

...

Some things that I know:

-My mac recently updated to Ventura 13.5.2
-I'm attempting to install Ruby v3.2.2
-I had been using Ruby v3.2.2 on my local machine before installing rbenv, which dropped it down to v3.1.2p20. 

Thanks in advance for the help, and let me know if anything is needed.

**Comment by dereknovak** [comment, marked as answer]
Issue resolved. I started from scratch, uninstalling and reinstalling Homebrew, then rbenv. Was able to successfully install Ruby 3.2.2, along with some earlier versions.

**Comment by serhiisvipe** [comment]
THANKS!!! Worked for me
